etc/vagrant/install.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+# Script to set up a Django project on Vagrant.
+PROJECT_NAME=$1
+
+DB_NAME=$PROJECT_NAME
+DB_USERNAME=$PROJECT_NAME
+DB_PASSWD=$PROJECT_NAME
+
+PROJECT_DIR=/home/vagrant/$PROJECT_NAME
+
+
+# Install essential packages from Apt
+apt-get update -y
+
+
+# Python dev packages
+apt-get build-dep -y python-imaging
+apt-get install -y python3-pip \
+        libjpeg8 \
+        libjpeg62-dev \
+        libfreetype6 \
+        libfreetype6-dev \
+        libpq-dev \
+        wget
+
+
+# Postgresql
+if ! command -v psql; then
+    apt-get install -y postgresql libpq-dev
+fi
+
+
+# install python packages
+su - -c "pip3 install -r $PROJECT_DIR/requirements/base.txt"
```

Comment:
BTW what's the reason for `su` here? You are calling `apt-get` few lines above without it